### PR TITLE
Carthage support

### PR DESCRIPTION
**Pull request by Lufthummel**
Added support for Carthage

### CODE REVIEW

**Comment by mmatoszko** [comment]
@mike4aday: any chance this could be merged? Works like a charm for me.

**Comment by mike4aday** [comment]
@mmatoszko Thanks - yes, I'll merge it. I've not used Carthage yet, so wanted to make sure I knew what I was merging. Give me a few days.

FYI @Lufthummel

**Comment by mmatoszko** [comment]
Thats great :)

File: Cartfile.resolved [hunk 1]
```diff
@@ -0,0 +1,3 @@
+github "Alamofire/Alamofire" "3.4.1"
+github "matthewpalmer/Locksmith" "2.0.8"
+github "mxcl/PromiseKit" "3.5.0"
```
Comment (mike4aday) [inline Cartfile.resolved]: Would Promise Kit version 3.2.1 work here also? I have the following in the Podspec: 
 s.dependency 'PromiseKit', '~> 3.2.1' 
To be consistent, the CocoaPods and Carthage dependencies should probably be the same -- I'll update the Podpsec if you cannot use the earlier version of PromiseKit for now.

**Comment by mike4aday** [comment]
@mmatoszko I decided to upgrade Swiftly Salesforce to Swift 3, which is in-progress, and then add Carthage support - so the Carthage support will take a little longer. 

Thanks.

**Comment by mike4aday** [comment]
Carthage support added today with release of version 3.3.2
FYI @Lufthummel @mmatoszko

**Pull request closed without merging**